cql3/statements/create_table_statement.cc

```diff
@@ -96,10 +97,30 @@ std::vector<column_definition> create_table_statement::get_columns()
     return column_defs;
 }
 
+template <typename CreateTable>
+future<shared_ptr<cql_transport::event::schema_change>>
+create_table_statement::create_table_with_cdc(service::storage_proxy& proxy,
```

Comment:
I'm curious why this is a separate function instead of just being an option inside the normal create table operation?

Reply:
I wanted to separate this CDC related logic to make it possible for a reader that's not interested in CDC to easily ignore this part.